Formula/ssh-vault.rb

```diff
@@ -0,0 +1,27 @@
+class SshVault < Formula
+  desc "SSH encrypt/decrypt using ssh keys"
+  homepage "https://ssh-vault.com/"
+  url "https://github.com/ssh-vault/ssh-vault.git",
+      :tag => "0.12.2",
+      :revision => "39c919577ab522293750d1c01174a8b5b9871aae"
+
+  head "https://github.com/ssh-vault/ssh-vault.git"
+
+  depends_on "go" => :build
+  depends_on "dep" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/ssh-vault/ssh-vault").install buildpath.children
+    cd "src/github.com/ssh-vault/ssh-vault" do
+      system "dep", "ensure"
+      ldflags = "-s -w -X main.version=#{version}"
+      system "go", "build", "-ldflags", ldflags, "-o", "#{bin}/ssh-vault", "cmd/ssh-vault/main.go"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system bin/"ssh-vault", "-v"
```

Comment:
Needs a "real" test that exercises and verifies the functionality of the software once installed.

Reply:
@ilovezfs  Hi, that's quite complex mainly because of how the package works: https://ssh-vault.com/post/how-it-works/

The main problem is that it will need a (private key) for decrypting something. 

Can the current test from the package work?

Reply:
@ilovezfs  I have been thinking on this,  and maybe the `-f` option could be used, it depends on one keys but in this case can be public, so basically the `-f` create the checksum on the key compatible with the default fingerprint that `AWS` uses.  I could use your for example:

    $ ssh-vault -u ilovezfs -f
    e4:1b:6d:bd:c9:5d:38:8a:20:58:1d:8a:d2:4a:e6:d7

And always verify it returns `e4:1b:6d:bd:c9:5d:38:8a:20:58:1d:8a:d2:4a:e6:d7` but for this you will not have to change your key, and well is another dependecy also.

Reply:
You might want to look at the `jose` formula for something similar to that.

Reply:
Hi @ilovezfs, wondering if there could be the please chance to accept the formula for now.

Also please let me ask again what is the issue because maybe I am miss understanding it.  

Do you mean something is wrong in:

    test do
        system bin/"ssh-vault", "-v" 

or you are asking me to create something more elaborated besides just printing the version of the package?

thanks in advance.

Reply:
>wondering if there could be please the chance to accept the formula for now.

We have a strict requirement that new formulae have a `test do` that verifies functionality.

>asking me to create something more elaborated besides just printing the version of the package

This. See `jose`, `jupyter`, and `faas-cli` for some good examples.

Reply:
@ilovezfs got it now and agree, by checking the example formulas that you just provided I got a better idea and created some tests.

Could you please check them, basically is doing the full encryption cycle. (encrypt/decrypt)

Reply:
@nbari Excellent work! I left a couple comments.